### ISSUE TITLE
authentication

### ISSUE THREAD

**Issue by asiripanich** [asiripanich/emdash#1]
Use [shinymanager](https://github.com/datastorm-open/shinymanager) or [shinyauthr](https://github.com/PaulC91/shinyauthr).

**Comment by shankari** [comment]
Looks like this is a prerequisite to deploying as part of a docker-compose stack. The good news is that in that case, you wouldn't need to use ssh to proxy the mongodb to your desktop.

**Comment by asiripanich** [comment]
Yes, that is the plan. But it takes quite a lot of resources to run. So I'm not sure if it is a good idea to include this in an emission docker-compose stack right now. Everyone probably wants to keep things as cheap as possible when they rent a cloud instance. :)
